triaging.rst

```diff
@@ -262,34 +332,74 @@ indicate a remote branch by adding ``#BRANCH`` to the end of the URL.
 
 Generating Special Links in a Comment
 -------------------------------------
-Comments can automatically generate a link to various web pages if formatted
-properly.
-
-* ``#<number>``, ``issue<number>``, or ``issue <number>`` links to the
-  tracker issue ``<number>``.
-* ``msg<number>`` links to the tracker message ``<number>``.
-* ``PR <number>``, ``PR<number>``, and ``pull request <number>`` can be used
-  to link to `GitHub pull requests <https://github.com/python/cpython/pulls>`_.
-* a 10-, 11-, 12-, or 40-digit hex ``<number>`` is assumed to be a Git or
-  Mercurial changeset identifier and generates a link to changeset ``<number>``
-  on GitHub or https://hg.python.org/.  The ``git`` and ``hg`` prefixes can
-  also be used to disambiguate, and must precede the number without spaces.
-* ``r<number>``, ``rev<number>``, or ``revision <number>`` is assumed to be
-  a legacy Subversion revision number, a reference to a changeset that was
-  checked in prior to 2011-03-05 when the official Python source code
-  repositories were migrated from the :abbr:`svn (Subversion)`
-  :abbr:`VCS (version control system)` to Mercurial.
-  The issue tracker automatically translates the legacy svn revision
-  ``<number>`` to its corresponding Mercurial changeset identifier.
-* ``Dir/file.ext`` and ``Dir/file.ext:NNN`` generate links to files in the
-  `Python source code repositories <https://github.com/python/cpython/>`_,
-  possibly linking to the line number specified after the ``:``.
-  ``3.6/Dir/file.ext`` will generate a link with ``3.6`` as branch.
-* ``PEP <number>`` and ``PEP<number>`` link to the
-  :abbr:`PEP (Python Enhancement Proposal)` ``<number>``.
-* ``devguide`` (lowercase), ``devguide/triaging``, and
-  ``devguide/triaging#generating-special-links-in-a-comment`` generate links to
-  the Devguide, this page, and this section respectively.
+Comment abbreviation can automatically generate a link to various web pages.
```

Comment:
Perhaps "Using the following abbreviations in a comment will automatically generate a link to relevant web pages."